src/app/games/coerceo/CoerceoPartSlice.ts

```diff
@@ -162,31 +162,40 @@ export class CoerceoPartSlice extends TriangularGameState {
         return true;
     }
     public isDeconnectable(tile: Coord): boolean {
-        const connectedSidesIndexes: number[] = this.getPresentNeighboorTilesIndexes(tile);
-        if (connectedSidesIndexes.length > 3) {
+        const neighboorsIndex: number[] = this.getPresentNeighboorTilesRelativeIndexes(tile);
+        if (neighboorsIndex.length > 3) {
             return false;
         }
+        let holeCount: number = 0;
         let i: number = 1;
-        while (i < connectedSidesIndexes.length) {
-            if (connectedSidesIndexes[i - 1] === connectedSidesIndexes[i] - 1 ||
-                (connectedSidesIndexes[0] === 0 && connectedSidesIndexes[connectedSidesIndexes.length - 1] === 5))
-            {
-                i++;
-            } else {
-                return false;
+        while (i < neighboorsIndex.length) {
```

Comment:
`for (let i: number = 1; i < connectedSidesIndexes.length; i++)` serait plus lisible non ?

Reply:
Va pour le for, quand aux connectedSides, neighboors est très très clair, vu qu'un voisin qui n'existe pas n'est pas vraiment un voisin ?